bitnami/mongodb/templates/replicaset/statefulset.yaml

```
@@ -501,6 +513,14 @@ spec:
             - key: mongodb-ca-key
               path: mongodb-ca-key
               mode: 0600
+        {{- else }}
+        {{- range $index, $secret := .Values.tls.replicaset.existingSecrets }}
+        - name: mongodb-certs-{{ $index }}
+          secret:
+            secretName: {{ tpl $secret $ }}
```

Same comment about the `tpl` function here:

```suggestion
            secretName: {{ include "common.tplvalues.render" ( dict "value" $secret "context" $) }}
```